plugins/woocommerce/src/Internal/Admin/Analytics.php

```diff
@@ -169,6 +175,27 @@ public function register_cache_clear_tool( $debug_tools ) {
 		return $debug_tools;
 	}
 
+	/**
+	 * Register the full refund fix data tool on the WooCommerce > Status > Tools page.
+	 *
+	 * @param array $debug_tools Available debug tool registrations.
+	 * @return array Filtered debug tool registrations.
+	 */
+	public function register_full_refund_fix_data_tool( $debug_tools ) {
+		if ( OrderUtil::uses_new_full_refund_data() ) {
+			return $debug_tools;
+		}
```

Comment:
On my local testing, [due to another error](https://github.com/woocommerce/woocommerce/pull/60814/files#r2335582612), the fix did not work, but the option to fix again was hidden because we deleted this option when scheduling the job - What I am thinking is, if due to timeouts/memory issues etc the fix does not work (Is this likely?) the option gets hidden.

Maybe we can check if there are any refunded orders in `wc_order_stats` but not in `wc_order_product_lookup`, and if there are still missing orders, we can show the button?

Reply:
Just read the reply here - https://github.com/woocommerce/woocommerce/pull/60814#discussion_r2333000451